crates/sui-move/src/manage_package.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use clap::Parser;
+use move_package::BuildConfig;
+use std::path::PathBuf;
+use sui_types::base_types::ObjectID;
+
+/// Record addresses (Object IDs) for where this package is published on chain (this command sets variables in Move.lock).
+#[derive(Parser)]
+#[group(id = "sui-move-manage-package")]
+pub struct ManagePackage {
+    #[clap(long)]
+    /// The network chain identifer. Use '35834a8a' for mainnet.
+    pub network: String,
+    #[clap(long = "original-id", value_parser = ObjectID::from_hex_literal)]
+    /// The original address (Object ID) where this package is published.
+    pub original_id: ObjectID,
+    #[clap(long = "latest-id", value_parser = ObjectID::from_hex_literal)]
+    /// The most recent address (Object ID) where this package is published. It is the same as 'original-id' if the package is immutable and published once. It is different from 'original-id' if the package has been upgraded to a different address.
+    pub latest_id: ObjectID,
+    #[clap(long = "version-number")]
+    /// The version number of the published package. It is '1' if the package is immutable and published once. It is some number greater than '1' if the package has been upgraded once or more.
+    pub version_number: u64,
+}
```

Comment:
All flags are required. As the doc comments suggest, in some cases values will be the same (e.g., `original-id` and `latest-id`). It's tempting to make some of these optional, but the scenarios I ran through all seemed to create a kind of footgun if we assume too much. 

E.g., if a user only specifies `original_id` we could assume `latest_id` is the same (so we make it optional) and `version_number` is 1 (so we make it optional). But then if `latest_id` to something other than `original_id`, `version_number` really shouldn't be optional. We could implement ad-hoc rules to try make this better too but again the benefit is marginal IMO, 4 required fields should be manageable for a user to deal with.

Reply:
Possibly nonobvious side note: `version_number` so as not to conflict with `--version` for the CLI command

Reply:
I think it's fine for everything to be required, because I expect in most cases, this will be interacted with programmatically.